mac/Keyman4MacIM/Keyman4MacIM/KMInfoWindow/KMInfoWindowController.m

```diff
@@ -82,7 +81,11 @@ - (BOOL)tabView:(NSTabView *)tabView shouldSelectTabViewItem:(NSTabViewItem *)ta
     return YES;
 }
 
+// TODO: refactor: any reason for this to be HTML? hard to read stringWithFormat applied to template with 16 arguments
```

Comment:
I think this was probably mimicking the Windows UX. The readme.htm file, if present, is currently presented in an iframe so would need to be included, and the QR code and image are also presented in HTML, so that would take additional rewriting.